runtime-modules/referendum/src/lib.rs

```diff
@@ -827,20 +828,25 @@ impl<T: Trait<I>, I: Instance> EnsureChecks<T, I> {
     }
 
     fn can_release_vote_stake(origin: T::Origin) -> Result<T::AccountId, Error<T, I>> {
-        let cycle_id = CurrentCycleId::<I>::get();
-
         // ensure superuser requested action
         let account_id = Self::ensure_regular_user(origin)?;
 
         let cast_vote = Self::ensure_vote_exists(&account_id)?;
 
+        // get current cycle id
+        let current_cycle_id = match Stage::<T, I>::get() {
+            ReferendumStage::Voting(stage_data) => Some(stage_data.current_cycle_id),
+            ReferendumStage::Revealing(stage_data) => Some(stage_data.current_cycle_id),
+            _ => None,
+        };
+
         // allow release only for past cycles
-        if cycle_id == cast_vote.cycle_id {
+        if current_cycle_id == Some(cast_vote.cycle_id) {
             return Err(Error::UnstakingVoteInSameCycle);
         }
 
         // ask runtime if stake can be released
-        if !T::can_unlock_vote_stake(&cast_vote, &cycle_id) {
+        if !T::can_unlock_vote_stake(&cast_vote) {
```

Comment:
This seems out of synch with signature on council, it accepts `current_voting_cycle_id: &u64`.

Reply:
This should have caused the CI to complain if I am correct, but its actually complaining for a differnt reason, so perhaps you can fix that formatting issue also.

Reply:
Unnecessary cycle id has been removed from the council function; synced now.